.gitignore

```diff
@@ -77,3 +77,9 @@ vagrant/puppet/modules/
 vagrant/puppet/.tmp
 .librarian
 .vagrant
+
+#Dirs created in dev
+OpenOversight/db_repository/
```

Comment:
So, I did this too, and on [reading the docs](https://flask-migrate.readthedocs.io/en/latest/) I'm not sure that we want to actually gitignore this. You've made changes to the declaration of the models, but I don't see any actual migrations generated, which leads me to believe that a) we are using flask-migrate wrong and b) the model changes below won't actually show up.